pom.xml

```diff
@@ -786,9 +786,9 @@
         <configuration>
           <errorRemedy>failFast</errorRemedy>
           <!--
-                               The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
-                              check your dependencies", verify the conditions of the license and add the reference to it here.
-                            -->
+                                         The list of allowed licenses. If you see the build failing due to "There are some forbidden licenses used, please
```

Comment:
nit: i believe that this change has happened in the past couple of releases, can we check that the formatter isn't doing anything strange?

Reply:
good question, I believe this is done by the plugin we use, I don't think the issue is in our logic

`mvn com.github.ekryd.sortpom:sortpom-maven-plugin:sort`